src/ansys/pre_commit_hooks/templates/README.md

```diff
@@ -0,0 +1,71 @@
+[![Python](https://img.shields.io/pypi/pyversions/{{ project_name }}?logo=pypi)](https://pypi.org/project/{{ project_name }}/)
+[![PyPI](https://img.shields.io/pypi/v/{{ project_name }}.svg?logo=python&logoColor=white)](https://pypi.org/project/{{ project_name }})
+[![Codecov](https://codecov.io/gh/ansys/{{ doc_repo_name }}/branch/main/graph/badge.svg)](https://codecov.io/gh/ansys/{{ doc_repo_name }})
+[![GH-CI](https://github.com/ansys/{{ doc_repo_name }}/actions/workflows/ci_cd.yml/badge.svg)](https://github.com/ansys/{{ doc_repo_name }}/actions/workflows/ci_cd.yml)
+[![MIT](https://img.shields.io/badge/License-MIT-yellow.svg)](https://opensource.org/licenses/MIT)
+[![Black](https://img.shields.io/badge/code%20style-black-000000.svg?style=flat)](https://github.com/psf/black)
+
+## Overview
+
+A Python wrapper for Ansys {{ product }}.
+
+## Installation
+
+You can use `pip <https://pypi.org/project/pip/>`_ to install {{ doc_repo_name }}.
+
+```sh
+  pip install {{ project_name }}
+```
+
+To install the latest development version, run these commands:
+
+```sh
+  git clone {{ repository_url }}
+  cd {{ doc_repo_name }}
+  pip install -e .
+```
+
+For more information, see [Getting Started](https://{{ product }}.docs.pyansys.com/version/stable/getting_started/index.html).
+
+## Basic usage
+
+This code shows how to import {{ doc_repo_name }} and use some basic capabilities:
+
+``` python
+    print("Put sample code here")
+```
+
+For comprehensive usage information, see [Examples](https://{{ product }}.docs.pyansys.com/version/stable/examples.html) in the [{{ doc_repo_name }} documentation](https://{{ product }}.docs.pyansys.com/version/stable/index.html).
+
+## Documentation and issues
+
+Documentation for the latest stable release of {{ doc_repo_name }} is hosted at [{{ doc_repo_name }} documentation](https://{{ product }}.docs.pyansys.com/version/stable/index.html).
+
+In the upper right corner of the documentation's title bar, there is an option for switching from
+viewing the documentation for the latest stable release to viewing the documentation for the
+development version or previously released versions.
+
+On the [{{ doc_repo_name }} Issues]({{ repository_url }}/issues) page,
+you can create issues to report bugs and request new features. On the
+[{{ doc_repo_name }} Discussions]({{ repository_url }}/discussions) page or
+the [Discussions](https://discuss.ansys.com/) page on the Ansys Developer portal,
+you can post questions, share ideas, and get community feedback.
+
+To reach the project support team, email [pyansys.core@ansys.com](mailto:pyansys.core@ansys.com).
+
+## Indices and tables
+
+-  [Getting Started](https://{{ product }}.docs.pyansys.com/version/stable/getting_started/index.html)
+-  [Examples](https://{{ product }}.docs.pyansys.com/version/stable/examples.html)
+-  [{{ doc_repo_name }} documentation](https://{{ product }}.docs.pyansys.com/version/stable/index.html)
```

Comment:
We could include this in the previous `Documentation` section together with a short explanation of which is the purpose of each one.